templates/review.pt.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "review"
+output: 
+  rmarkdown::md_document:
+    pandoc_args: [
+      "--wrap=none"
+    ]
+---
+
+## Revisão do pacote
+
+*Marque as caixas conforme aplicável e elabore nos comentários abaixo. Sua avaliação não se limita a estes tópicos, conforme descrito no guia do(a) revisor(a)*
+
+- **Descreva resumidamente qualquer relacionamento de trabalho que você tem (teve) com as pessoas autoras do pacote.**
+- [ ] Como revisor(a), confirmo que não há [conflitos de interesse](https://devguide.ropensci.org/policies.html#coi) para que eu revise este trabalho (se você não tiver certeza se está em conflito, fale com seu editor(a) _antes_ de iniciar sua revisão)
+
+#### Documentação
+
+O pacote inclui todas as seguintes formas de documentação:
+
+- [ ] **Uma declaração de necessidade:** declarando claramente os problemas que o software foi projetado para resolver além do seu público-alvo no README
+- [ ] **Instruções de instalação:** para a versão de desenvolvimento do pacote e quaisquer dependências que sejam fora do padrão no README
+- [ ] **Vignette(s):** demonstrando as principais funcionalidades com exemplos que são executados localmente com sucesso
+- [ ] **Documentação de funções:** para todas as funções exportadas
+- [ ] **Exemplos:** (que são executados localmente com sucesso) para todas as funções exportadas
+- [ ] **Diretrizes da comunidade:** incluindo diretrizes de contribuição no README ou CONTRIBUTING, e DESCRIPTION com `URL`, `BugReports` e `Maintainer` (o qual pode ser gerado automaticamente via `Authors@R`).
+
+#### Funcionalidade
+
+- [ ] **Instalação:** Processo de instalação documentado conclui com sucesso.
+- [ ] **Funcionalidade:** Qualquer funcionalidade que foi assumida pelo software foi confirmada.
+- [ ] **Performance:** Qualquer performance a mais que foi assumida pelo software foi confirmada.
```

Comment:
Although "performance" is an English word commonly used in Portuguese due to the common anglicism in our language, we have its equivalent "desempenho" which fits perfectly. So here is my suggestion to use it instead.

```suggestion
- [ ] **Desempenho:** Qualquer desempenho a mais que foi assumida pelo software foi confirmada.
```

Reply:
```suggestion
- [ ] **Desempenho:** Qualquer desempenho a mais que foi assumido pelo software foi confirmado.
```

Reply:
Right, thanks for adjusting the genre, @pedropark99 !